src/path.cc

```diff
@@ -240,8 +240,7 @@ std::string PathResolve(Environment* env,
   const size_t numArgs = paths.size();
 
   for (int i = numArgs - 1; i >= -1 && !resolvedAbsolute; i--) {
-    const std::string& path =
-        (i >= 0) ? std::string(paths[i]) : env->GetCwd(env->exec_path());
+    const std::string& path = (i >= 0) ? std::string(paths[i]) : cwd;
```

Comment:
Why this is unnecessary? AFAIK we call it on win32

Reply:
I think it's unnecessary because it's populated on line 239?